tsl/src/remote/dist_ddl.c

```diff
@@ -116,6 +116,19 @@ dist_ddl_error_if_not_allowed_data_node_session(void)
 	dist_ddl_error_raise_blocked();
 }
 
+static void
+dist_ddl_error_if_multi_command(const ProcessUtilityArgs *args)
+{
+	List *parsetree_list;
+
+	/* Parse the SQL string into a list of raw parse trees */
+	parsetree_list = pg_parse_query(args->query_string);
+
+	/* Prevent 'command;command' execution */
+	if (list_length(parsetree_list) != 1)
+		dist_ddl_error_raise_unsupported();
```

Comment:
`ERROR:  operation not supported on distributed hypertable` sounds misleading. ANALYZE is supported, multi-commands aren't.

Reply:
Why not say "multiple commands are not supported on distributed tables"? That would be the correct description of the limitation we have. If I run "analyze this; analyze that" and it says "the operation is not supported", the operation here is analyze. That's just wrong.

Reply:
I would assume the operation here is the supplied query text to the server. Also saying `multiple commands are not supported on distributed hypertable` is also a bit misleading, because they are supported. Even multi-statement transactions are supported. There are some exceptions of course, like you cannot mix local and distributed hypertables, etc. The problem mostly, because we do not rewrite DDL queries (with some exceptions).

Yet I understand your point, messaging could be better. Let me change it